core/db_client.py

```diff
@@ -0,0 +1,53 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import json
+import pandas as pd
+
+
+class DBClient(object):
```

Comment:
Please write a docstring for the class.

Also, please put a note that authentication is not implemented yet so at the moment any data provider can change the labels for any data provider. We need to fix this later.

Reply:
Added docstring.

Could you clarify what you mean by:

> any data provider can change the labels for any data provider

We manage access to DB, and each data provider can only see the data they're giving us...?